gen/com/goide/psi/GoImportSpec.java

```diff
@@ -51,4 +51,6 @@
 
   boolean isCImport();
 
+  boolean isSDKImport();
```

Comment:
I'll need to reread the GoPathScopeHelper again as I don't understand how it can be used to check if an import refers to the Sdk or not.